.eslintrc.cjs

```diff
@@ -121,6 +121,17 @@ module.exports = {
         '@jessie.js/safe-await-separator': 'off',
       },
     },
+    {
+      // These tests use EV() instead of E(), which are easy to confuse.
+      // Help by erroring when E() packages are imported.
```

Comment:
I'm a little surprsed we don't use `E` at all in these tests. And not sure we will never want to. But I guess we can revisit should a good reason arise.